src/pip/_internal/vcs/subversion.py

```diff
@@ -286,34 +288,41 @@ def export(self, location, url):
                 # Subversion doesn't like to check out over an existing
                 # directory --force fixes this, but was only added in svn 1.5
                 rmtree(location)
-            cmd_args = (['export'] + self.get_remote_call_options() +
-                        rev_options.to_args() + [url, location])
+            cmd_args = make_command(
+                'export', self.get_remote_call_options(),
```

Comment:
I’m not a big fan of the one argument per line style honestly because it adds so much excess vertical space. It makes it harder to read IMO because it leads to having to scroll a lot more to see a function’s implementation. It also makes text searches less informative because you only see the one word in the search result, i.e. no surrounding context.